frontend/src/pages/Traces/CustomColumns/columns.ts

```diff
@@ -48,6 +48,14 @@ const SECURE_SESSION_ID_COLUMN: TraceCustomColumn = {
 	accessKey: 'secureSessionID',
 }
 
+const HAS_ERRORS_COLUMN: TraceCustomColumn = {
+	id: 'has_errors',
+	label: 'Has Errors',
+	type: 'boolean',
+	size: '1fr',
+	accessKey: 'hasErrors',
+}
```

Comment:
Agree with your comment about displaying this in a better way in the future. I think we could show whether a trace has errors in a subtle but obvious way rather than giving it an entire column. Do we have a ticket for that?